modules/ROOT/pages/vpc-provisioning-concept.adoc

```diff
@@ -79,6 +79,16 @@ You can create your Anypoint VPC inside any of the available CloudHub regions. +
 The recommended region to use might vary depending on how you connect to your Anypoint VPC. If you are using a VPN tunnel, you might want to choose the CloudHub region closest to your data center. +
 However, if you are peering with your private AWS VPC, you need to create your Anypoint VPC in the same AWS region.
 
+== FAQ: How to deploy or move applications to a VPC
```

Comment:
Is it common for us to use FAQ headings?
Is it OK that this is not a question, in that case?

Reply:
It's not really unless it's in an actual FAQ topic. But this is legacy content and there are already several headings like this. I think all the FAQs should be broken out into an FAQ topic, but that was beyond the scope of this PR.